UnityProject/Assets/Scripts/Items/Attributes.cs

```diff
@@ -274,4 +281,23 @@ public void ServerSetArticleDescription(string desc)
 	{
 		SyncArticleDescription(articleDescription, desc);
 	}
+
+	public List<string> SearchableString()
+	{
+		var names = new List<string>();
+		names.Add(initialName);
+		names.Add(articleName);
+		if (this is ItemAttributesV2 c)
+		{
+			names.AddRange(c.GetTraits().Select(trait => trait.name));
+		}
+		return names;
+	}
+
+	public void HighlightObject()
+	{
+		if (PlayerManager.LocalPlayerObject == null || gameObject.IsAtHiddenPos()) return;
+		if (Vector2.Distance(gameObject.AssumedWorldPosServer(), PlayerManager.LocalPlayerObject.AssumedWorldPosServer()) > MINIMUM_HIGHLIGHT_DISTANCE) return;
+		Instantiate(GlobalHighlighterManager.HighlightObject, this.transform, false);
```

Comment:
when does the highlight go away? I see no code to unhighlight the object.

Reply:
It unhighlights it self from the other script

Reply:
what other script?